gradle/dependency-versions.gradle

```diff
@@ -1,5 +1,5 @@
 ext {
-    LIKafkaVersion = "1.0.41"
+    LIKafkaVersion = "1.0.52"
```

Comment:
@somandal this was not part of the original commit that you're reverting. Just checking if it's intentional.

Reply:
Yes this is intentional. Kafka reverted the changes on their end in this version. I just want to bump this up for any OSS users that might be using passthrough so that they don't run into a performance regression.